global_oce_cs32/README.md

```diff
@@ -8,17 +7,55 @@ git clone https://github.com/MITgcm/verification_other
 cd verification_other/global_oce_cs32
 wget ftp://mit.ecco-group.org/ecco_for_las/version_4/checkpoints/core2_cnyf.tar
 tar xf core2_cnyf.tar
-\rm -f core2_cnyf.tar
+# in case you want to save some space:
+#rm core2_cnyf.tar
+```
+
+# compile the code
+
+[Note that building the AD-executable requires access to [TAF](http://www.fastopt.de).]
+
+The following steps are basically what `../verification/testreport -t global_oce_cs32 -adm` does if executed in directory `verification_other`.
 
-ln -s ../verification_other/global_oce_cs32 .
 ```
+cd build
+ ../../../tools/genmake2 -mods ../code_ad ${-of your_build_options_file} ${other options, e.g. -nocat4ad}
+make CLEAN
+make depend
+make
+
+```
+
+The final `make` command will build `mitgcmuv` for forward integrations. For the AD-executable `mitgcmuv_ad`, you have to do
+
+```
+make adall
+```
+
+which will call `taf` (after you have paid for it) to generate the AD-code and then compile.
```

Comment:
suggest replacing "(after you have paid for it)" with the need to get license for TAF with a link to FastOpt